tests/sentry/integrations/slack/test_utils.py

```diff
@@ -103,3 +103,193 @@ def test_rate_limiting(self):
         )
         with pytest.raises(ApiRateLimitedError):
             get_channel_id(self.organization, self.integration, "@user")
+
+
+class GetChannelIdFasterTest(TestCase):
+    @with_feature("organizations:slack-use-new-lookup")
+    def setUp(self):
+        self.resp = responses.mock
+        self.resp.__enter__()
+
+        self.integration = install_slack(self.event.project.organization)
+        self.add_list_response(
+            "conversations",
+            [
+                {"name": "my-channel", "id": "m-c"},
+                {"name": "other-chann", "id": "o-c"},
+                {"name": "my-private-channel", "id": "m-p-c", "is_private": True},
+            ],
+            result_name="channels",
+        )
+        self.add_list_response(
```

Comment:
You can move this to `test_invalid_member_selected_display_name` which I think is the only test that needs it and doesn't have it.